hack/update-openapi-spec.sh

```diff
@@ -83,7 +84,7 @@ fi
 
 kube::log::status "Updating " ${OPENAPI_ROOT_DIR}
 
-curl -w "\n" -fs "${API_HOST}:${API_PORT}/openapi/v2" > "${OPENAPI_ROOT_DIR}/swagger.json"
+curl -w "\n" -fs "${API_HOST}:${API_PORT}/openapi/v2" | jq . > "${OPENAPI_ROOT_DIR}/swagger.json"
```

Comment:
this unescapes `<` and `>` chars in output... not necessarily a problem, just noting it

Reply:
I this file more than documentation? Where is it actually used?

Reply:
it gets loaded as a fixture by some kubectl tests, otherwise mostly as documentation

Reply:
the checked-in spec is also used for generating [clients](https://github.com/kubernetes-client). the generator loads the spec from release branch

Reply:
To be clear, it's still valid JSON and a valid spec, just doesn't have HTML-escaped angle brackets